core/arch/arm/kernel/boot.c

```diff
@@ -416,8 +416,9 @@ static void init_runtime(unsigned long pageable_part)
 
 	init_asan();
 
-	malloc_add_pool(__heap1_start, __heap1_end - __heap1_start);
+
```

Comment:
A short comment on this order would be nice.

Reply:
ok